cs251tk/student/markdownify/process_file.py

```diff
@@ -133,8 +133,15 @@ def process_file(filename, *, steps, options, spec, cwd, supporting_dir, interac
     if not should_continue or skip_web_compile and options['web']:
         return results
 
-    should_continue = compile_file(filename, steps, results, supporting_dir,
-                                   basedir, web, student, spec_id)
+    should_continue = compile_file(filename,
+                                   steps=steps,
+                                   results=results,
+                                   supporting_dir=supporting_dir,
+                                   basedir=basedir,
+                                   web=web,
```

Comment:
… what's the difference between the argument `web` and `options['web']`?

Reply:
Web is if the --web flag is used, options['web'] is if the file is marked with web: true

Reply:
Ah, ok. Thanks for clearing that up.